src/Auth0/Login/LoginServiceProvider.php

```diff
@@ -21,11 +21,11 @@ class LoginServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        \Auth::provider('auth0', function ($app, array $config) {
+        \Illuminate\Support\Facades\Auth::provider('auth0', function ($app, array $config) {
```

Comment:
Would it be possible to import these with `use` statements at the top of the file as opposed to going the inline route? Not sure if this was a technical requirement, just curious if that would be a cleaner approach otherwise.